### ISSUE TITLE
the Foundation: A repository of human knowledge

### ISSUE THREAD

**Issue by TheDataDesigner** [district0x/district-proposals#241]
## **Purpose**: 
Use the power of a decentralized community to maintain an immutable record of human history and a sharable library of knowledge that cannot be censored or deleted.

This would prevent political administrations from tampering with scientific data that they disagree with, make it easier to combat propaganda and fake news, give people access to a wealth of information and knowledge, and allow people to profit from their experiences and expertise by contributing to the collective knowledge of our species.

## **Description**: 
Foundation is organized into 2 sections: Encyclopedia and Timeline. Entries can and should link to other entries within and between sections.

### ***Encyclopedia***
A repository of formatted data, information, and knowledge maintained by the community. Think wikipedia, but with more emphasis on author reputation, source authenticity, the ability to more formally structure data for human and machine use, along crypto-powered incentives to maintain quality and accuracy. The goal is to create a platform that allows anyone to share their knowledge and experience in ways that can benefit others.

Anyone should be able to contribute to the Encyclopedia, from historians writing an analysis of a historical event and scientists publishing their experiment process and data for others to confirm and use, to a handyman sharing a videos on household maintenance and bloggers posting healthy recipes.

Authors can choose to release their content for free or license its use. Licensing could be conditional based on the consumer as well. For instance, a resource could be free for students, but paid for companies. Or a resource could be free to humans, but paid for bots. We could also look into some kind of professional subscription model, where a subscription gets you access to everyone, or a everything within a topic, and revenue is distributed to all of the authors of resources that the subscriber consumed over the course of the subscription period.
If authors charge users, a small portion of the revenue will be diverted and added to the “Prosperity Pool” which is a fund used to support community endeavors and incentivize users to help curate and maintain the free content. The proportion of licensing fees allocated to this pool is one of the items that would be decided by the community.

### ***Timeline***
A chronological sequence of world events collectively agreed upon as accurate and true. Similar to the district proposal process in District0x, users can propose events by staking tokens, challenge others events, vote and contribute to reach a consensus on a fair and accurate depiction of world events, then publish them to the blockchain. Unlike Encyclopedia entries, all timeline data is free and accessible to everyone.

Users are incentivized to contribute to and maintain this timeline through rewards, either from newly minted Foundation tokens (like miners in Bitcoin) or through the Posterity Pool. Oracles could also be created to automate the process of codifying external data sources.

## **Ethereum Address**:
0x936ef46690b36d19c67839e21f8fe59562467e4b

**Comment by Bradymck** [comment]
This is awesome. Is this something you are currently working on already? I would love to see what you have so far.

**Comment by Bradymck** [comment]
I can't think of a good tag for this proposal off the top of my head. Would you like me to create one so others can easily find it? I'm all ears for suggestions.

**Comment by Bradymck** [comment]
I will leave untagged for now. I like when I get a new proposal that requires a new category. As far as further developing the idea, I think this is a really important one but other proposals are in the lead so making a mockup or two would go a long way in getting people interested enough to vote.

**Comment by TheDataDesigner** [comment]
Thanks for the tip. I'll work on putting some visuals together.

**Comment by TheDataDesigner** [comment]
I've been working hard to keep this moving, so I wanted to share an update. I've created a [repo](https://github.com/TheDataDesigner/Foundation) for this project, and expanded on the original proposal by starting a draft of the [white paper](https://github.com/TheDataDesigner/Foundation/blob/master/Documents/white-paper.md).

**Comment by Bradymck** [comment]
Hey just a thought, I was curious if you see journalism playing a role in the timeline. There is a few good ones I think you may find interesting: 

https://github.com/district0x/district-proposals/issues/70
https://github.com/district0x/district-proposals/issues/107

I think there is room for collaboration here, what do you think?

**Comment by TheDataDesigner** [comment]
Absolutely, adoption of Foundation by journalists as a trusted source is a perfect use case. There is a lot of opportunities for collaboration! 

A first I thought that in addition to the Timeline and Encyclopedia, there could be a newsroom at some point for a deeper dive into the events recorded to Foundation. Perhaps it would be a better approach to simply make data on Foundation accessible through an API so that people, including journalists, can embed Foundation data directly into 3rd party platforms. A less technical solution is to make it dead simple to cite resources within Foundation, like a button that copies a formatted citation to the users clipboard.

**Comment by Bradymck** [comment]
Hey Alex, I wanted to meet back up with you at the event to get you a shirt. Since we didn't can you DM me your address? I'll add you to the distribution list.

**Comment by Bradymck** [comment]
Hey @TheDataDesigner 


Don't forget about the new bounty to migrate your proposals to the actual District Registry: https://registry.district0x.io/

We're replacing the old voting app with the registry. Let me know if you need help but I would love to see you migrate this over so you can claim your DNT.

It does take a 10,000 DNT submit to submit your proposal but this gives you an extra 2000 you can stake in the registry beyond the deposit amount.

Hit me up on Telegram or Discord if you need help or have questions.

Telegram: https://t.me/district0x/75217
Discord: https://discord.gg/P9RQejv

PS, please excuse the canned response. I am encouraging everyone here to start migrating so they can claim their 12000 DNT.

Also, this type of "immutable wikipedia" is in dire need right now.
